ci-operator/config/openshift/node-observability-operator/openshift-node-observability-operator-main.yaml

```diff
@@ -1,8 +1,46 @@
+base_images:
+  base:
+    name: "4.10"
+    namespace: ocp
+    tag: base
+  kube-rbac-proxy:
+    name: "4.10"
+    namespace: ocp
+    tag: kube-rbac-proxy
+  ocp_builder_rhel-8-golang-1.17-openshift-4.10:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.17-openshift-4.10
+  operator-index:
+    name: redhat-operator-index
+    namespace: ci
+    tag: v4.9
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
     tag: golang-1.17
+images:
+- dockerfile_path: Dockerfile.rhel8
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.17-openshift-4.10:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10
+  to: node-observability-operator
+operator:
+  bundles:
+  - as: node-observability-operator-bundle
+    base_index: operator-index
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/openshift/node-observability-operator:v0.0.1
```

Comment:
@sherine-k - This is great , I see you picked this up and it then gets substituted - nice